sweep.py

```diff
@@ -128,13 +129,7 @@ def Bs(ix):
 
 #Apply boundary conditions, allow sliding along a vertical axis in the middle
 ix_i = int(nDOF/3/2)*3 #The middle vertical DOF
-ix = range(ix_i+1,ix_i+3)
-K = np.delete(K, ix, axis=0)
-K = np.delete(K, ix, axis=1)
-M = np.delete(M, ix, axis=0)
-M = np.delete(M, ix, axis=1)
-F = np.delete(F, ix, axis=0)
-F = np.delete(F, ix, axis=1)
+ix = range(ix_i,ix_i+3)
```

Comment:
The location external force applying.
When we mistake the choich of these three components in Force,
Chladni patterns may break symmtery.
(Is the order of Force components are u,v,w? It's not like...)